app/javascript/packs/internal.tsx

```diff
@@ -1,6 +1,7 @@
 import 'easymde/dist/easymde.min.css'
 
 import React, { lazy, Suspense } from 'react'
+import currency from 'currency.js'
```

Comment:
@dem4ron can we remove this from application.tsx now?

Reply:
Done